doc/lua_api.txt

```diff
@@ -4094,6 +4120,24 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Uses texture (string)
         playername = "singleplayer"
     --  ^ optional, if specified spawns particle only on the player's client
+	material_type_param = minetest.pack_texture_blend_func(
+			minetest.ebf.src_alpha, minetest.ebf.one,minetest.emfn.modulate_1x, 
+			minetest.eas.texture + minetest.eas.vertex_color),
```

Comment:
I prefer not. Examples are important for understanding of what those variables actually doing.